src/community/blocks.tsx

```diff
@@ -0,0 +1,347 @@
+
+    secondHeading?: {
+        value: string;
+        text: string;
+    };
+
+    secondaryValue?: string;
+
+    title?: string;
+    description?: string | Array<string>;
+
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    tag,
+    mainHeading,
+    secondHeading,
+    secondaryValue,
+    title,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            {tag && (
+                <div
+                    className={sprinkles({
+                        paddingBottom: 24,
+                    })}
+                >
+                    {tag}
+                </div>
+            )}
+
+            <Stack space={2}>
+                <Inline space={8} alignItems="baseline">
+                    <Text8 color={valueColor}>{mainHeading.value}</Text8>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {mainHeading.text}
+                    </Text2>
+                </Inline>
+                {secondHeading && (
+                    <Inline space={8} alignItems="baseline">
+                        <Text8 color={valueColor}>{secondHeading.value}</Text8>
+                        <Text2 regular color={vars.colors.textSecondary}>
+                            {secondHeading.text}
+                        </Text2>
+                    </Inline>
+                )}
+                {secondaryValue && <Text8 color={vars.colors.textSecondary}>{secondaryValue}</Text8>}
+            </Stack>
+            <div
+                className={sprinkles({
+                    paddingTop: 8,
+                })}
+            >
+                <BlockContent title={title} description={description} />
+            </div>
+        </div>
+    );
+};
+
+interface ValueBlockProps {
+    title?: string;
+    value?: string;
+    description?: string | Array<string>;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const ValueBlock: React.FC<ValueBlockProps> = ({
+    title,
+    value,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            aria-label={ariaLabel}
+            style={{
+                display: 'flex',
+                flexDirection: 'column',
+            }}
+        >
+            <Text2 regular color={vars.colors.textPrimary}>
+                {title}
+            </Text2>
+            <Text8 color={valueColor}>{value}</Text8>
+            <BlockContent description={description} />
+        </div>
+    );
+};
+
+interface ProgressBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+
+    progressPercent?: number;
+    reverse?: boolean;
+
+    value: string;
+    text: string;
+    description?: string;
+
+    valueColor?: string;
+    'aria-label'?: string;
```

Comment:
I changed the value and text to be passed together in an object, in the example of figma it seems that the heading is the only one required, the example of use cases are in figma -> Slots - Props

I also changed value to accept number as well as string

Reply:
It doesn't make sense to have a `ProgressBlock` without a `ProgresBar` If figma specs say so, then figma specs must be wrong.

Why did you changed  value to accept number? if it should only accept numbers, then type it as `number`, if it should accept any string then type it as `string` but don't type it as `number | string`, please

Reply:
It happens that there are cases that do not have consumption, so there is this possibility that the progressBar does not appear.

Already for value, I will change then to accept only number

Reply:
Then, what's the difference between a ProgressBlock without progress and a HighlightedValueBlock?
```ts
<ProgressBlock heading={{value: 34, text: 'some text'}} description="some description"/>
<HighlightedValueBlock mainHeading={{value: 34, text: 'some text'}} description="some description" />
```
![image](https://github.com/Telefonica/mistica-web/assets/1849576/d8e7de9f-f828-4a5a-84d2-1d90f89a6b27)

Reply:
In fact, you can customize them in this way where there is no difference between them, because the blocks were thought to have great versatility due to several business rules.

The difference is that each one has its own resources that would allow a more unique use, for example in the one you placed each one would have the possibility of using its own props like the HighlightedValueBlock being able to use the tags while progressBlock a stackingGroup

Reply:
I don't like the idea of having two different block components that can be used to render the same design